care/emr/resources/report/template/spec.py

```
@@ -0,0 +1,114 @@
+            msg = f"Invalid template_type '{v}'. Valid types are: {', '.join(valid_types)}"
+            raise ValueError(msg)
+        return v
+
+
+class TemplateCreateSpec(TemplateBaseSpec):
+    facility: UUID4 | None = None
+    slug_value: SlugType
+    context_config: dict = {}
+    template_data: str
+
+    def perform_extra_deserialization(self, is_update, obj):
```

Ruff (0.14.4)

Unused method argument: `is_update`

(ARG002)